streampark-console/streampark-console-webapp/src/views/flink/setting/View.vue

```diff
@@ -676,7 +676,7 @@
             type="text"
             allowClear
             placeholder="Please enter alert name"
-            v-decorator="['alertName',{ rules: [{validator: handleCheckAlertName,required: true , message: 'Alert Name is required'}]} ]"/>
+            v-decorator="['alertName',{ rules: [{required: true , message: 'Alert Name is required'},{validator: handleCheckAlertName}]} ]"/>
```

Comment:
recommend this way:  `{ rules: [{ validator: handleCheckAlertName,trigger:'submit',required: true } ]}`

Reply:
OK, suggestion has been accepted and i has update this pr,PTAL